app/src/components/Fields/ZipField.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextFieldMaterial from '@material-ui/core/TextField';
+import MaskedInput from 'react-text-mask';
+
+function TextMaskCustom(props) {
+  const { inputRef, ...other } = props;
+
+  return (
+    <MaskedInput
+      {...other}
+      ref={ref => {
+        inputRef(ref ? ref.inputElement : null);
+      }}
+      guide
+      mask={[/[1-9]/, /[1-9]/, /[1-9]/, /[1-9]/, /[1-9]/]}
```

Comment:
you can replace this with /^[1-9]{5}/

Reply:
@danmelton yeah, it should, but within `React Text Mask` it seems to need it on a character by character basis the way I did it.